Enterprise-CMCS/macpro-mdct-mfp
# Update ui-src unit tests

### Description

- Fix tests that are causing console errors
- Reorganize tests under a single `describe` in each file with clearer descriptions
  - For component tests, use `<ComponentName />` as the top level description
  - For methods, use `methodName()` as the description
- Use `test` instead of `it` for test declarations
- Addresses tests not updated in https://github.com/Enterprise-CMCS/macpro-mdct-mfp/pull/676

### Related ticket(s)


CMDCT-3778

---
### How to test

1. cd to `macpro-mdct-mfp/services/ui-src`
2. Run `yarn coverage`
3. Verify tests pass and coverage is unaffected
4. Verify console errors are gone

![Screenshot 2024-06-27 at 11 43 37 AM](https://github.com/Enterprise-CMCS/macpro-mdct-mfp/assets/169298580/cd996ea3-63f7-4bca-ad2b-cf32b5be29cf)

### Notes


Example Test Output:
  - Current test descriptions:
![Screenshot 2024-06-27 at 12 11 40 PM](https://github.com/Enterprise-CMCS/macpro-mdct-mfp/assets/169298580/bc14b02d-d8e5-44b6-be0f-fffec82845f9)
  - After organizing: 
![Screenshot 2024-06-27 at 12 12 55 PM](https://github.com/Enterprise-CMCS/macpro-mdct-mfp/assets/169298580/eae61ea3-f32d-4f98-bdc4-64dc1f69c5e2)
---

### Pre-review checklist
